### PR TITLE
Fix space-between bug by adding empty view as fillers

### PR DESCRIPTION
**Pull request by VansonLeung**
Fix space-between bug by adding empty view as fillers

### CODE REVIEW

File: PhotoGrid.js [hunk 1]
```diff
@@ -60,6 +60,13 @@ class PhotoGrid extends React.Component {
     let itemWidth = Math.floor( (deviceWidth - totalMargin) / itemsPerRow );
     let adjustedMargin = ( deviceWidth - (itemsPerRow*itemWidth) ) / (itemsPerRow - 1);
 
+    let emptyViewsCount = (this.props.itemsPerRow || 3) - items.length;
+    var emptyViews = [];
+    for (var i = 0; i < emptyViewsCount; i++)
+    {
+      emptyViews.push(<View style={{width: itemWidth}}/>);
```
Comment (worldlyjohn) [inline PhotoGrid.js]: where are you printing out `emptyViews`?

Comment (worldlyjohn) [inline PhotoGrid.js:67]: you probably need `key={i}`

File: PhotoGrid.js [hunk 2]
```diff
@@ -60,6 +60,13 @@ class PhotoGrid extends React.Component {
...
+    }
+
     return (
       <View style = {[ styles.row, { marginBottom: adjustedMargin } ]}>
         { items.map(item => this.props.renderItem(item, itemWidth)) }
```
Comment (worldlyjohn) [inline PhotoGrid.js:72]: think you forgot: 
`{ emptyViews.map((item, idx) => item) }`

**Comment by christopherabouabdo** [comment]
Closing this because it never got finished up and another PR was merged that resolves the same issue

**Pull request closed without merging**